src/blueprints/tv2/action-factories/tv2-replay-action-factory.ts

```diff
@@ -74,14 +74,14 @@ export class Tv2ReplayActionFactory {
   }
```

Comment:
This should use `.replaceAll`. 

When using a string for the pattern in the `.replace`, only the first occurrence is changed. 
If a regex is used instead in `.replace` then it updates all occurrences.

Reply:
Hmm, it didn't seem to want to put my comment on the correct line.
It was suppose to be on the line for `return value.replace(' ', '')` just above...

Reply:
Updated